src/pages/Campaign/CampaignListAndSearch.tsx

```diff
@@ -46,49 +47,55 @@ export default function CampaignListAndSearch({
       <CampaignList>
         {filteredCampaigns.map((campaign, index) => {
           const isSelected = selectedCampaign && selectedCampaign.id === campaign.id
+          const isRewardInUSD = campaign.rewardDistribution[0].rewardInUSD
 
           const totalRewardAmount: Fraction = campaign.rewardDistribution.reduce((acc, value) => {
             return acc.add(
               new Fraction(
                 JSBI.BigInt(value.amount ?? '0'),
-                JSBI.exponentiate(JSBI.BigInt(10), JSBI.BigInt(value?.token?.decimals ?? 18)),
+                isRewardInUSD ? ONE : JSBI.exponentiate(JSBI.BigInt(10), JSBI.BigInt(value?.token?.decimals ?? 18)),
               ),
             )
           }, new Fraction(0))
 
+          const rCampaignName = campaign.name
+          const rCampaignStatus =
+            campaign.status === 'Upcoming' ? t`Upcoming` : campaign.status === 'Ongoing' ? t`Ongoing` : t`Ended`
+          const rChainIdImages =
+            campaign &&
+            campaign.chainIds &&
+            campaign.chainIds
+              .split(',')
+              .map(chainId => (
+                <img
+                  key={chainId}
+                  src={
+                    isDarkMode && !!NETWORKS_INFO[chainId as unknown as ChainId].iconDark
+                      ? NETWORKS_INFO[chainId as unknown as ChainId].iconDark
+                      : NETWORKS_INFO[chainId as unknown as ChainId].icon
+                  }
+                  alt="network_icon"
```

Comment:
Should we provide more details? Like ``{NETWORKS_INFO[chainId as unknown as ChainId].name + ' icon'}``

Reply:
done